### DISCUSSION TITLE
Get metadata of URLs

### DISCUSSION THREAD

**Discussion by purnimaax** [explosion/spaCy#13268] [Help: Coding & Implementations]
So I have an excel sheet with many website URLs and my task is to find the following about the URLs and the data present in those websites:
Industry (the text speaks about)
Country/Region
Campaign Type: E-commerce, Lead Generation, etc.
Content Type: Blog/Article, Research/White paper, Case Study, Industry Report, etc.
Platform: Google, Facebook, Instagram, LinkedIn, etc.
Ad Type: Display, Video, App, OTT, etc.

If we are not able to find few or any of the above about a particular URL, we can simply put N/A. But the main focus is on the accuracy.
Which model or API should I use?

**Comment by Sirivasv** [comment]
You can take a look at the spaCy universe page to get some models or APIs:
https://spacy.io/universe

Perhaps something like:
https://spacy.io/universe/project/conciseconcepts where you can define a starting `data` as:
```
data = {
    "Industry": ["space", "motor", "fashion"],
    "Country": ["Spain", "Germany", "France"],
...
}
```
So that it can give you with a given probability which concepts were found in a text from a site. 

Other projects in the spacy.io/universe might be helpful, or even a conversational one could be of help as in passing the contents from the link to a chatbot and ask for that metadata.
